lib/tests/test_conflicts.rs

```diff
@@ -1027,26 +1036,6 @@ fn test_parse_conflict_simple() {
     )
     "#
     );
-    // The conflict markers are too long and shouldn't parse (though we may
```

Comment:
Should we keep this or do we have new tests that cover the case of parsing a conflict that's longer than necessary for no good reason?

Reply:
I think it would be good to keep it; I'll add it back.